bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/immutable/transform/UpConverter.scala

```diff
@@ -58,7 +58,7 @@ object UpConverter {
     case p @ <proposal>{ns @ _*}</proposal> if (p \ "@schemaVersion").text == Proposal.currentSchemaVersion =>
       StepResult(Nil, node).successNel[String]
     case p @ <proposal>{ns @ _*}</proposal> if (p \ "@schemaVersion").text == "2018.2.1"                    =>
-      from2018A1.concatenate.convert(node)
+      from2018B.concatenate.convert(node)
```

Comment:
I'm not sure how this works, but wouldn't you still need the 2018A1 upgrade to happen?   Or is that subsumed in the new converter?

Reply:
It should be subsumed in the new converter, but I see that @cquiroz added the `from2018A1` just last week. I'm not sure what the purpose of it is, since I can't see anything it is doing. @cquiroz, could you help?